src/libs/settings.js

```diff
@@ -9,24 +9,49 @@ export async function getSettings() {
 		//  Enable the logging automatically if we cannot determine the user preference.
 		settings = {};
```

Comment:
doesn't this need the { proplate: {}, hoverCard: {} } form to prevent js errors further down?
maybe using the defaultOptions() or should the all fall back to false like now?

btw: naming convention: settings or options? and maybe refactor the defaultOptions() to be here in settings.js

Reply:
```suggestion
		settings = {
			proplate: {},
			hoverCard: {},
		};
```
how does the suggestion thingie work again... anyway good catch, ty :3
---
the naming convention is like that on purpose. options is everything directly attached to the popup window - the html, css and js. As such I don't think i can easily move `defaultOptions()` into settings.js, since that is a library for the big protoots.js script.
maybe we can differentiate them in a less confusing way?